index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Calculator</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+    <div class="calc">
+        <section class="screen">0</section>
```

Comment:
I would rename this to "calc-screen" so we follow the same naming convention as the buttons below.